webapp/templates/team/problems.html.twig

```diff
@@ -7,6 +7,7 @@
         problem_text_path: 'team_problem_text',
         problem_sample_testcase_path: 'team_problem_sample_testcase',
         problem_attachment_path: 'team_problem_attachment',
-        problem_sample_zip_path: 'team_problem_sample_zip'
+        problem_sample_zip_path: 'team_problem_sample_zip',
+        show_submit_button: true
```

Comment:
Why is this always true? Should it only be true when the contest is started and not ended?

Reply:
It is only true for teams, not for public (which reuses the same Twig snippet that we include).
The whole problemset page is not shown if the contest is not running (and if you go there you get an empty page)

Reply:
Fair enough